### ISSUE TITLE
Use highest temperature rather than average

### ISSUE THREAD

**Issue by Redsandro** [efluffy/gpdfand#5]
Rather than using the average in [gpdfand.pl:119](https://github.com/efluffy/gpdfand/blob/master/gpdfand.pl#L119):

```
        foreach (@temps) {
            $average += $temps[$counter];
            $counter++;
        }
        $average = $average/$counter;
```

Since Atom Z8750 has a single core burst mode of 2.5 GHz, perhaps it's better to use the core with the highest temperature?

E.g. `$temp = max @temps` (Guessed syntax. Not familiar with perl.)

**Comment by efluffy** [comment]
this is a good idea. the original version did check cores independently but there was no reasoning behind that logic -- it's just how i wrote it. at the time, everyone was still running linux on the gpd with cpu turbo disabled, so averaging was fine, but now that turbo is working perhaps this should be re-done.

**Comment by efluffy** [comment]
added in cf0344e69f419101c792c46db01b55c2ae79e9c6

test if you're able, I won't have access to my GPD for a week or so, can't test myself right now.

if it works well I'll close this.

**Comment by Redsandro** [comment]
It works. I cleaned up the code and used utils for simpler code:

Pull Request: https://github.com/efluffy/gpdfand/pull/6

**Comment by efluffy** [comment]
merged #6
